src/app/components_ngrx/iteration-list-entry/iteration-list-entry.component.less

```diff
@@ -14,6 +14,13 @@
       width: 15px;
       padding: 5px;
       font-size: 16px;
+      &:hover {
+        color: @color-pf-blue;
+        cursor: pointer;
```

Comment:
@SMahil @debloper why do we have to be so detailed about everything? Shouldn't a link automatically get a pointy cursor?

Reply:
@kwk It should be but right now it's not the case. I can plan a thorough check to fix this kind of issues.

Reply:
@kwk yes, it automatically becomes pointer (unless forced not to) if we're talking about the same anchor element. However, here the style is getting applied on a div element, and the anchor is a different child element. Due to classic over-nesting of DOM in bootstrap framework that we tend to get used to, and carry it around - situations like this happen (and is often overlooked).

@SMahil I've verified this in my PR (where the DOM is restructured to a leaner functional alternative with less container nesting), if you set the main clickable object itself as anchor tag (the way it should be), then it won't be necessary to explicitly set cursors with CSS.

Reply:
@kwk @debloper this is for icon not for the iteration name block